balena.env

```diff
@@ -0,0 +1,4 @@
+
+export BASE_URL=balena.io
```

Comment:
The API base URL is actually `balena-cloud.com`

Reply:
nice catch, thank you @pimterry!